distribution/docs/src/main/resources/content/_installing/solr-service.adoc

```diff
@@ -0,0 +1,72 @@
+:title: Run Solr as Managed Service
+:type: startingIntro
+:status: published
+:summary: Install Solr as a Service.
+:project: ${branding}
+:order: 05
+
+==== {title}
+
+These instructions are configuring Solr as a service managed by the operating system. A common
+reason is to start Solr when the system boots.
+
+===== Configure Solr as a Systemd Service
+These instructions are for the unix family of operating systems running the systemd
```

Comment:
✏️ Unix?

Reply:
I had to google this one.  I found different advice, but this made the most sense to me:
http://www.greens.org/about/unix.html
@ricklarsen What would you like to see?

Reply:
I think the wording is okay, but shouldn't Unix have a capital 'U'?